gui/src/2D/atlasSourceImage.ts

```diff
@@ -0,0 +1,37 @@
+module BABYLON.GUI {
+    /**
+     * Defines the interface used storage information atlas texture
+     */
+    export interface IAtlasSourceData {
+        frame: any,
+        pivot: any,
+        rotated: boolean,
+        sourceSize: any,
+        spriteSourceSize: any,
+        trimmed: boolean,
+    }
+
+    /**

```

Comment:
Class used to store an element of an Atlas texture
May be renamed it to AtlasElementSourceImage